docs/user-guide/time-and-costs/progress-tracking/README.md

```diff
@@ -89,15 +145,29 @@ In Status-based progress reporting mode, Work is not a required value. However,
 
 This allows you to edit %&nbsp;Complete (by changing status) or Work and get a preview of the updated Remaining work before saving changes.
 
->**Note:** In the upcoming version, statuses cannot have an empty %&nbsp;Complete value in status-based progress reporting mode.When upgrading, all statuses that do not have a value will take the default value of 0%.
+> [!NOTE]
+> In the upcoming version, statuses cannot have an empty %&nbsp;Complete value in status-based progress reporting mode.When upgrading, all statuses that do not have a value will take the default value of 0%.
 
 ## Hierarchy totals
 
 OpenProject will automatically show totals for Work, Remaining work and % Complete in a work package hierarchy (any parent with children). These appear in a work package table as a number with a Σ sign next to it, indicating that it is a total of the values of the parent _and_ children.
 
 ![Hierarchy totals for Work, Remaining work and % Complete](hierarchy-totals-days.png)
 
-> **Note**: The total %&nbsp;Complete value of a hierarchy is a weighted average tied to Work. For example, a feature with Work set to 50h that is 30% done will influence the total of %&nbsp;Complete of the parent more than a feature with Work set to 5h that is 70% done.
+OpenProject offers two modes for calculating *% Complete* in hierarchy totals:
+
+- **Weighted by work**: The total %&nbsp;Complete value of a hierarchy is a weighted average tied to Work. For example, a feature with Work set to 50h that is 30% done will influence the total of %&nbsp;Complete of the parent more than a feature with Work set to 5h that is 70% done. 
+
+  > [!TIP]
+  >
+  > Work packages without *Work* will be ignored.
+
+- **Simple average**: *Work* is ignored and the *total % Complete* is calculated as a simple average of the *% Complete* values from the work packages in the hierarchy.
```

Comment:
actually, it's not exactly true: it's an average of direct children only, not an average of all descendants. And if a child has children itself, then we take its total % complete value instead of the % complete value for the computation.

```suggestion
- **Simple average**: *Work* is ignored and the *total % Complete* is calculated as a simple average of the *% Complete* values from the direct work packages children in the hierarchy. The value used on each direct child for computing the average is its *total % Complete* value, or its *% Complete* value if it has no children, or 0% if its *% Complete* value is not set.
```